docs/integrations/kubeflow.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: Kubeflow
+description: Easily build reproducible data pipelines with Kubeflow and lakeFS using commits, without modifying the code or logic of your job. 
+parent: Integrations
+nav_order: 56
+has_children: false
+
+---
+
+
+[Kubeflow](https://www.kubeflow.org/docs/about/kubeflow/) is a project dedicated to making deployments of ML workflows on Kubernetes simple, portable and scalable.
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC 
+{:toc}
+
+## Using lakeFS with Kubeflow pipelines
+{: .no_toc }
+A Kubeflow pipeline is a portable and scalable definition of an ML workflow composed of steps. Each step in the pipeline is an instance of a component represented as an instance of [ContainerOp](https://kubeflow-pipelines.readthedocs.io/en/latest/source/kfp.dsl.html#kfp.dsl.ContainerOp).
+
+There are some aspects we will need to handle to run Kubeflow with lakeFS:
+## Install lakeFS CLI 
+lakeFS comes with its own native CLI client. We will use it to build lakeFS-specific Kubeflow pipeline steps. 
+To install and configure the lakeFS CLI client follow [this](https://docs.lakefs.io/quickstart/lakefs_cli.html) guide.  
+
+## Add lakeFS-specific pipeline steps
+Once you have `lakectl` installed, you can add pipeline components that invoke lakeFS operations. As mentioned above, a pipeline component is represented as a ContainerOp.
+For `lakectl` to work with Kubeflow, you will need to provide your `lakectl` configurations as environment variables named:
```

Comment:
Do I need those even if I follow the documentation on how to configure lakectl, using for example `lakectl config`? The relation between this paragraph and the previous one (titled "Install lakeFS CLI") is not clear.

Reply:
In this example, we use env variables to pass the lakefs configuration.
1. the end-user can use volume and mount our lakectl yaml configuration (but we don't need to explain everything) it is not just a must
2. we should look into how to pass secrets to the operation

Reply:
Following @johnnyaug 's question, I changed the structure of the docs. can you give it another look and let me know your thoughts? 

@nopcoder 
1. I added a note mentioning this.. let me know if you think this is redundant
2. 100% agree, IMO we should do this when we develop an official proper Kubeflow integration. WDYT?